src/QuantumSdk/DefaultItems/DefaultItems.targets

```diff
@@ -0,0 +1,101 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Work around issues in MSBuild versions prior to 16.0. -->
+  <PropertyGroup>
+    <MSBuildAllProjects Condition=" '$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0' ">
+      $(MSBuildAllProjects);$(MSBuildThisFileFullPath)
+    </MSBuildAllProjects>
+  </PropertyGroup>
+  
+  <!-- We want access to the full path of packages as determined by msbuild to filter and reorder various package references. -->
+  <ItemDefinitionGroup>
+    <PackageReference GeneratePathProperty="true" IsQscReference ="false" />
+  </ItemDefinitionGroup>
+  
+
+  <!-- Packages containing rewrite steps that are executed during compilation. -->
+  <ItemGroup>
+    <!-- for backwards compatibility we still generate C# independent on whether the project is targeted for execution on the simulation framework or not -->
+    <PackageReference 
+      Include="Microsoft.Quantum.CsharpGeneration" Version="0.10.1911.1607" IsImplicitlyDefined="true" 
+      IsQscReference ="true" ExecutionTarget="Any" SdkInternalStage="AfterTargeting" /> <!-- Todo: we will want to make the order of execution properly configurable -->
+  </ItemGroup>
+
+  <!-- Packages and libraries included for all execution targets. -->
+  <ItemGroup>
+    <!-- TODO: Core & Intrinsics package -->
+    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.10.1911.1607" IsImplicitlyDefined="true" />
+  </ItemGroup>
+
+
+  <!-- Includes packages and project references depending on the defined project properties. -->
+  <Target Name="PropertyDependentReferences" DependsOnTargets="QsharpPropertyResolution" BeforeTargets="ResolveAssemblyReferences">
+    <ItemGroup>
+      <!-- Packages included for specific execution targets. -->
+      <PackageReference Condition="'$(ResolvedQsharpExecutionTarget)' == 'SimulatorBackend'" Include="Microsoft.Quantum.Simulators" Version="0.10.1911.1607" IsImplicitlyDefined="true" />
+    </ItemGroup>
+  </Target>   
+
+
+  <!-- Resolution of verified configurable properties. -->
+  <Target Name="QsharpPropertyResolution">
+    <!-- resolving the OutputType to either QsharpExe or QsharpLibrary -->
+    <PropertyGroup>
+      <ResolvedQsharpOutputType Condition="'$(OutputType)' == 'Library'">QsharpLibrary</ResolvedQsharpOutputType>
```

Comment:
```suggestion
      <ResolvedQsharpOutputType Condition="'$(OutputType)' == 'Library'">QSharpLibrary</ResolvedQsharpOutputType>
```

Reply:
So, we currently have one property QsharpLangVersion that is in used in our internal tools and checked by the language server. I'd rather not break the name of that property. If we want to leave that, then it makes sense to me to at least be consistent and stick with the lowercase s all the way through. I take is CSharp/FSharp chooses capitalization in properties?